### DISCUSSION TITLE
On sabbatical and can't verify location

### DISCUSSION THREAD

**Discussion by josephausterweil** [community/community#49165] [GitHub Education]
Hello,

I'm trying to renew my teacher status. I'm in Japan on sabbatical and so when it checks my location, it notes that my university does not have any schools in my location (which is the case -- I'm at University of Wisconsin - Madison). 

Could you please fix this situation? 

Even on sabbatical I am advising students and actively using GitHub as a means for teaching them version control.

Thanks

**Comment by PaulWAyers** [comment]
I have the same issue. I even tried using the campus VPN but GitHub seems to see right through it....
